### PR TITLE
Exception errors to stderr

### LINKED ISSUE #5207: stderr not used by Conan
At least, the exception messages printed in the `command.py` should go to stderr. This is important to improve the CI automation feedback on errors.

Maybe we should have the stderr stream on the conan output and print there also the .error messages

### PR DESCRIPTION
**Pull request by lasote**
Changelog: Feature: Print errors and warnings to `stderr`
Docs: omit

Close #5207

### CODE REVIEW

**Comment by lasote** [comment]
This fix is very partial, probably we should manage in out outputer both streams and print the errors to stderr also.

File: conans/client/command.py [hunk 1]
```diff
@@ -1711,19 +1711,23 @@ def run(self, *args):
...
+                sys.stderr.write("Exiting with code: %d" % exc.code)
             ret_code = exc.code
         except ConanInvalidConfiguration as exc:
             ret_code = ERROR_INVALID_CONFIGURATION
             self._user_io.out.error(exc)
+            sys.stderr.write(str(exc))
         except ConanException as exc:
             ret_code = ERROR_GENERAL
             self._user_io.out.error(exc)
+            sys.stderr.write(str(exc))
```
Comment (memsharded) [inline conans/client/command.py]: This duplicates errors in users output:
```bash
$ conan install zlib/1.2.11@conan/stable -s os=AIX
ERROR: Invalid setting 'AIX' is not a valid 'settings.os' value.
Possible values are ['Android', 'Arduino', 'FreeBSD', 'Linux', 'Macos', 'SunOS', 'Windows', 'WindowsStore', 'iOS', 'tvOS', 'watchOS']
Read "http://docs.conan.io/en/latest/faq/troubleshooting.html#error-invalid-setting"
Invalid setting 'AIX' is not a valid 'settings.os' value.
Possible values are ['Android', 'Arduino', 'FreeBSD', 'Linux', 'Macos', 'SunOS', 'Windows', 'WindowsStore', 'iOS', 'tvOS', 'watchOS']
Read "http://docs.conan.io/en/latest/faq/troubleshooting.html#error-invalid-setting"
```

Comment (lasote) [reply conans/client/command.py]: So then we have to do it correctly managing both streams in the conan ouput.

Comment (lasote) [reply conans/client/command.py]: I'll update

File: conans/client/output.py [hunk 2]
```diff
@@ -86,7 +87,10 @@ def write(self, data, front=None, back=None, newline=False):
         # Windows output locks produce IOErrors
         for _ in range(3):
             try:
-                self._stream.write(data)
+                if error:
```
Comment (memsharded) [inline conans/client/output.py]: The opposite?

Comment (lasote) [reply conans/client/output.py]: yes, that explains some funny things I'm experiencing.

**Comment by lasote** [comment]
Looking at the stderr I see messages like:

```
zlib/1.2.11@conan/stable: WARN: Forced build from source
zlib/1.2.11@conan/stable: WARN: Building...
```
Should they be considered warnings?

**Pull request merged**